music21/metadata/properties.py

```diff
@@ -0,0 +1,1732 @@
+    #   the resource since its creation that are significant for its
+    #   authenticity, integrity, and interpretation.
+    PropertyDescription(
+        name='provenance',
+        namespace='dcterms',
+        isContributor=False),
+
+    # publisher: An entity responsible for making the resource available.
+    PropertyDescription(
+        name='publisher',
+        namespace='dcterms',
+        valueType=Contributor,
+        isContributor=True),
+
+    # references: A related resource that is referenced, cited, or
+    #   otherwise pointed to by the described resource.
+    PropertyDescription(
+        name='references',
+        namespace='dcterms',
+        isContributor=False),
+
+    # relation: A related resource.
+    PropertyDescription(
+        name='relation',
+        namespace='dcterms',
+        isContributor=False),
+
+    # replaces: A related resource that is supplanted, displaced, or
+    #   superseded by the described resource.
+    PropertyDescription(
+        name='replaces',
+        namespace='dcterms',
+        isContributor=False),
+
+    # requires: A related resource that is required by the described
+    #   resource to support its function, delivery, or coherence.
+    PropertyDescription(
+        name='requires',
+        namespace='dcterms',
+        isContributor=False),
+
+    # rights: Information about rights held in and over the resource.
+    PropertyDescription(
+        name='rights',
+        namespace='dcterms',
+        oldMusic21WorkId='copyright',
+        valueType=Copyright,
+        isContributor=False),
+
+    # rightsHolder: A person or organization owning or managing rights
+    #   over the resource.
+    PropertyDescription(
+        name='rightsHolder',
+        namespace='dcterms',
+        valueType=Contributor,
+        isContributor=True),
+
+    # source: A related resource from which the described resource
+    #   is derived.
+    PropertyDescription(
+        name='source',
+        namespace='dcterms',
+        isContributor=False),
+
+    # spatial: Spatial characteristics of the resource.
+    PropertyDescription(
+        name='spatial',
+        namespace='dcterms',
+        uniqueName='spatialCoverage',
+        isContributor=False),
+
+    # subject: A topic of the resource.
+    PropertyDescription(
+        name='subject',
+        namespace='dcterms',
+        isContributor=False),
+
+    # tableOfContents: A list of subunits of the resource.
+    PropertyDescription(
+        name='tableOfContents',
+        namespace='dcterms',
+        isContributor=False),
+
+    # temporal: Temporal characteristics of the resource.
+    PropertyDescription(
+        name='temporal',
+        namespace='dcterms',
+        uniqueName='temporalCoverage',
+        isContributor=False),
+
+    # title: A name given to the resource.
+    PropertyDescription(
+        name='title',
+        namespace='dcterms',
+        oldMusic21Abbrev='otl',
+        oldMusic21WorkId='title',
```

Comment:
is this needed if it's the same as current name?

Reply:
It used to be, when I was being all backward compatible, and I really needed to know the full list of workIds (for example, all() was only returning workIds, not anything new).  I could go through and only put oldMusic21WorkId when it's different.

Reply:
I did that.